templates/security/cis.html

```diff
@@ -79,8 +99,11 @@ <h2>Which versions of Ubuntu have CIS tooling?</h2>
       </tbody>
     </table>
     <p>
-      <a href="/support#get-in-touch" class="p-button--positive">Access the Ubuntu CIS benchmark tooling</a>
-      <a href="https://security-certs.docs.ubuntu.com/en/cis" class="p-button--neutral p-link--external">Learn more about our tools</a>
+      <a href="/security/certifications#get-in-touch" class="p-button--positive">Access the Ubuntu CIS benchmark tooling</a>
```

Comment:
This should be a modal I think?
```suggestion
      <a href="/security/certifications#get-in-touch" class="p-button--positive js-invoke-modal">Access the Ubuntu CIS benchmark tooling</a>
```